default.css

```diff
@@ -146,9 +130,15 @@ select {
 }
 
 .f-col {
+  display: flex;
   flex-direction: column;
 }
 
+.f-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
by default the direction is row. unless you want to override a column direction, you can avoid it

Reply:
Thankyou for pointing this out!

Reply:
noted